### PR TITLE
Initial selenium grid test

### LINKED ISSUE #7192: Create a customer focused selenium grid application test
Our customers are starting to use our product in various interesting ways, one such way is selenium grid applications. We need to create a test that exercises VIC using selenium grid application in a customer like environment in order to feel more confident that our customers will be successful.

Should be straightforward in our nightlies. Just create the proper environment in nimbus and then install VIC and run selenium compose file with about 30 selenium minions. Getting to that point is probably good enough, doing some simple selenium tests might be beneficial but not necessary I think as long as we confirm that the minions are running and didn't throw any errors during the test.

### PR DESCRIPTION
**Pull request by mhagen-vmware**
fixes #7192

### CODE REVIEW

File: tests/manual-test-cases/Group5-Functional-Tests/5-27-Selenium-Grid.robot [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+# limitations under the License
+
+*** Settings ***
+Documentation  Test 5-27 - Selenium Grid
+Resource  ../../resources/Util.robot
+Suite Setup  Wait Until Keyword Succeeds  10x  10m  Selenium Grid Create
+Suite Teardown  Run Keyword And Ignore Error  Nimbus Cleanup  ${list}
+
+*** Keywords ***
+Selenium Grid Create
```
Comment (mdharamadas1) [inline tests/manual-test-cases/Group5-Functional-Tests/5-27-Selenium-Grid.robot]: I think this keyword naming is inappropriate as its creating a testbed and not selenium grid itself?

File: tests/manual-test-cases/Group5-Functional-Tests/5-27-Selenium-Grid.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+# References:
+[1 - Selenium Grid](https://github.com/SeleniumHQ/docker-selenium/blob/master/README.md)
+
+
+# Environment:
+This test requires access to VMware Nimbus cluster for dynamic ESXi and vCenter creation
+
+# Test Steps:
+1. Deploy a new vCenter with a simple cluster
+2. Create a docker network for the selenium grid
```
Comment (rajanashok) [inline tests/manual-test-cases/Group5-Functional-Tests/5-27-Selenium-Grid.md]: lgtm, May be I am missing it, where in the workflow docker network is created for the grid?

File: tests/manual-test-cases/Group5-Functional-Tests/5-27-Selenium-Grid.robot [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+    Fail  Selenium node ${node-name} failed to start properly
+
+*** Test Cases ***    
+Test
+    Log To Console  Starting Selenium Grid test...
+    Custom Testbed Keepalive  /dbc/pa-dbc1111/mhagen
+
+    Install VIC Appliance To Test Server
+
+    ${rc}  ${output}=  Run And Return Rc And Output  docker %{VCH-PARAMS} network create grid
```
Comment (mhagen-vmware) [inline tests/manual-test-cases/Group5-Functional-Tests/5-27-Selenium-Grid.robot]: @rajanashok here

Comment (rajanashok) [inline tests/manual-test-cases/Group5-Functional-Tests/5-27-Selenium-Grid.robot]: Found it, never mind.

**Pull request merged**